### ISSUE TITLE
PointNet++, number of points in custom dataset 

### ISSUE THREAD

**Issue by lmlepin9** [rusty1s/pytorch_geometric#1688]
## ❓ Questions & Help

Hi, first I would like to thank you for the amazing work done in torch.geometric. 

I'm trying to train PointNet++ on a custom data set for multilabel classification. The set is composed of images obtained from a simulated 3D pixel readout. There are different shapes simulated within a box, so the point clouds contain x,y,z coordinates, and pixel intensities. Each element from this set contains a different amount of points. I've been following the example1 structure. I just modified the size of the first MLP to accept the 4 dimensions of my point clouds. 

My questions are:

Is it necessary to have a fixed amount of points? For instance, in the example1 it is fixed to 1024
Is it necessary to center and normalize the point cloud values? 

Thanks for your time.

Best,

LM

**Comment by rusty1s** [comment]
Our PointNet++ implementation can handle varying sizes of point clouds. In general, it is not required to center and normalize the point clouds, but it may help to improve generalization. This is something that just needs to be evaluated to see which performs better.

**Comment by lmlepin9** [comment]
Great, thanks a lot!
